apps/transport/test/transport_web/plugs/worker_healthcheck_test.exs

```diff
@@ -0,0 +1,128 @@
+defmodule TransportWeb.Plugs.WorkerHealthcheckTest do
+  # async: false is required because we use real in-memory caching in these tests
```

Comment:
```suggestion
  # async: false is required because we use real in-memory caching in these tests,
  # and we swap application config (shared state)
```